versionRuleset.xml

```diff
@@ -0,0 +1,50 @@
+<ruleset comparisonMethod="maven"
+         xmlns="http://mojo.codehaus.org/versions-maven-plugin/rule/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://mojo.codehaus.org/versions-maven-plugin/rule/2.0.0 https://www.mojohaus.org/versions-maven-plugin/xsd/rule-2.0.0.xsd">
+<rules>
+    <rule groupId="*" artifactId="*" comparisonMethod="maven">
+      <!--
+            This regex is intended to restrict version updates to the last stable releases of dependencies and plugins.
+
+            Any version that does not contain the stable maven-plugin version naming scheme (EG "-M7"),
+                but does contain letters a-zA-Z (and optional digits after)
+       -->
+       <ignoreVersions>
+          <ignoreVersion type="regex">^(?!.[\s\S]*-M\d*).[\s\S]*.*[a-zA-Z]\d.*$</ignoreVersion>
```

Comment:
Sometimes when I try to be clever it works against me, but this seemed to help remove unwanted beta updates from the previously listed maven commands in the pom.

Please feel free to verify this configuration.  If comment this out and run the version update commands you should see additional content for 'alpha' version of slf4j-api
```
[INFO] The following dependencies in Dependencies have newer versions:
[INFO]   org.slf4j:slf4j-api ........................... 1.7.36 -> 2.0.0-alpha7
```
That version is not an official release, though it is newer.

I think it would be good to occassionally disable this rule just to verify that it's working the way it's intended.  Currently it's working the best I was able to configure for the situation.